### DISCUSSION TITLE
best practice for pop up ?

### DISCUSSION THREAD

**Discussion by LuckyTurtleDev** [hecrj/iced#1194] [Q&A] [answered]
What is the best practice for pop up?
An additional sandbox? Or exist their something easier?

As example I would like to create a pup up, which does print the error message of an Result.

A option to create popups as example for a question or a simple warning/information, without creating a new sandbox for each would be nice, too.

**Comment by 13r0ck** [comment, marked as answer]
The [Iced_aw](https://github.com/iced-rs/iced_aw) modal widget

**Reply by LuckyTurtleDev** [reply]
I have start using [native-dialog](https://crates.io/crates/native-dialog), but I will also checkout iced_aw.
Thanks

**Reply by Barugon** [reply]
Thanks for pointing this out. I've been thinking about doing file dialog like [this](https://github.com/Barugon/egui_file) but for Iced.
